### PR TITLE
Cust 2335 as v2 dev i expect to give title for the location picker

### PR DESCRIPTION
**Pull request by rwrzesien**
This should implement:
https://peachworks.leankit.com/Boards/View/267929177/275459230

### CODE REVIEW

File: src/directives/location-alert/location-alert.spec.js [hunk 1]
```diff
@@ -49,13 +49,8 @@ describe('Directive: Message', () => {
...
     let element = $compile('<peach-location-alert></peach-location-alert>')($rootScope);
 
-    $rootScope.$digest();
-
-    let stubCleanup = sandbox.stub($rootScope.$root.locationAlert, 'cleanup').returns(true);
-
-    $rootScope.$emit('$destroy');
-
-    sinon.assert.calledOnce(stubCleanup);
+    var isolateScope = angular.element(element).eq(0).scope();
```
Comment (jsmylnycky) [inline src/directives/location-alert/location-alert.spec.js]: this should be 'let'

Comment (rwrzesien) [inline src/directives/location-alert/location-alert.spec.js]: @jsmylnycky 'let' fixed, the purpose of this test is to cover, I didn't find it possible to stub controller function from directive context.

Comment (jsmylnycky) [inline src/directives/location-alert/location-alert.spec.js]: @rwrzesien  You should be able to put a stub on your "scope.cleanup()" function and make sure it gets called.

Comment (rwrzesien) [inline src/directives/location-alert/location-alert.spec.js]: @jsmylnycky Yes, but this scope is not accessible from test.

**Comment by jsmylnycky** [comment]
@rwrzesien  I fixed the unit tests for you since I dont want this held up and your attention is on Panera.  Please review the changes I put in to actually test so you understand how to do it in the future :)

**Comment by jsmylnycky** [comment]
:+1:

**Comment by rynomars** [comment]
:+1:

**Comment by jsmylnycky** [comment]
@rwrzesien  please merge squash

**Comment by rwrzesien** [comment]
@jsmylnycky Merged and pushed, 7db3da1cb59d11908fe62ed50592fc724264c25a

**Comment by jsmylnycky** [comment]
v0.7.12

**Pull request closed without merging**